Etabs_Adapter/CRUD/Create/SectionProperty.cs

```diff
@@ -238,5 +241,167 @@ private void SetProfile(IProfile profile, string sectionName, IMaterialFragment
+                                                EnumToString(EtabsSettings.DatabaseSettings.SectionDatabase),
+                                                m_DBSectionsNames[index]))
+            {
+                return false;
+            }
+
+            Engine.Reflection.Compute.RecordNote(bhSection.Name + " properties has been assigned from the database section " + bhName + ".");
+            return true;
+        }
+
+        /***************************************************/
+
+        private Dictionary<string, string> GetDictionary(SectionDatabase sectionDatabase)
+        {
+            switch (sectionDatabase)
+            {
+                case SectionDatabase.BSShapes2006:
+                    return new Dictionary<string, string>()
+                    {
+                        { "UB",  "UKB" },
+                        { "UC",  "UKC" },
+                        { "UBP", "UKBP" },
+                        { "L",   "UKA" },
+                        { "PFC", "UKPFC" },
+                        { "CHS", "CHHF" },
+                        { "RHS", "RHHF" },
+                        { "SHS", "SHHF" },
+                        { "TUB", "UKT" },
+                        { "TUC", "UKT" }
+                    };
+                case SectionDatabase.None:
+                case SectionDatabase.AISC14:
+                case SectionDatabase.AISC14M:
+                case SectionDatabase.AISC15:
+                case SectionDatabase.AISC15M:
+                case SectionDatabase.ArcelorMittal_British:
+                case SectionDatabase.ArcelorMittal_BritishHISTAR:
+                case SectionDatabase.ArcelorMittal_Europe:
+                case SectionDatabase.ArcelorMittal_EuropeHISTAR:
+                case SectionDatabase.ArcelorMittal_Japan:
+                case SectionDatabase.ArcelorMittal_Russia:
+                case SectionDatabase.ArcelorMittal_US_ASTM_A913:
+                case SectionDatabase.ArcelorMittal_US_ASTM_A913M:
+                case SectionDatabase.ArcelorMittal_US_ASTM_A992:
+                case SectionDatabase.ArcelorMittal_US_ASTM_A992M:
+                case SectionDatabase.Australia_NewZealand:
+                case SectionDatabase.ChineseGB08:
+                case SectionDatabase.CISC9:
+                case SectionDatabase.CISC10:
+                case SectionDatabase.CoreBraceBRB_2016:
+                case SectionDatabase.Euro:
+                case SectionDatabase.Indian:
+                case SectionDatabase.JIS_G_3192_2014:
+                case SectionDatabase.Nordic:
+                case SectionDatabase.Russian:
+                case SectionDatabase.SJIJoists:
+                case SectionDatabase.StarSeismicBRB:
+                default:
+                    return new Dictionary<string, string>();
+            }
+        }
+
+        /***************************************************/
+
+        private string EnumToString(SectionDatabase sectionDB)
```

Comment:
Again, would change this method name to something a bit more explicit, like

`ToEtabsFileName` for example